tests/FakeItEasy.IntegrationTests/Assertions/ExceptionMessagesTests.cs

```diff
@@ -4,13 +4,13 @@ namespace FakeItEasy.IntegrationTests.Assertions
     using FakeItEasy.Tests;
     using FakeItEasy.Tests.TestHelpers;
     using FluentAssertions;
-    using NUnit.Framework;
+    using Xunit;
+    using Record = FakeItEasy.Tests.TestHelpers.Record;

```

Comment:
Shall we kill `FakeItEasy.Tests.TestHelpers.Record` in this changeset? Presumably it's now a redundant duplication?